channels/cl.m3u

```diff
@@ -143,3 +143,5 @@ http://cdn.streamingmedia.cl:1935/live/vtvvina/playlist.m3u8
 https://tls.cdnz.cl/xtvmusic/xtvmusic/playlist.m3u8
 #EXTINF:-1 tvg-id="ZappingMusicHD.cl" tvg-name="Zapping Music HD" tvg-country="CL" tvg-language="" tvg-logo="https://i.imgur.com/lHvoKdw.jpg" group-title="Music",Zapping Music HD
 https://zmlive.zappingtv.com/zm_free/zm.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="CL" tvg-language="" tvg-logo="https://graph.facebook.com/KdeTelevision/picture?width=300&height=300" group-title="",Kanade TV (KDE)
+https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.twitch.tv/kdetelevision
```

Comment:
Offline
![image](https://user-images.githubusercontent.com/7253922/120027217-c489e500-bffb-11eb-8a22-1ffb8f32689c.png)

Reply:
@LaneSh4d0w The website https://query-streamlink.lanesh4d0w.repl.co/ don't working right now.

Reply:
Dang, it seems it had cut off again. 2 seconds.

Reply:
It's booting back up. It normally goes back automatically but it seems it didn't this time.

Reply:
@Nintendocustom 's implementation might be added in case mine doesn't go as properly.

https://iptv--iptv.repl.co/streamlink?url=https://www.twitch.tv/kdetelevision

Reply:
Thank you, the link works again.

Reply:
@freearhey Could you review this link again?